tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/composite_instruction.h

```diff
@@ -239,8 +269,32 @@ class CompositeInstruction
    * If not provided, the planner should use the current state of the robot is used and defined as fixed.
    */
   value_type start_instruction_{ NullInstruction() };
+
+public:
+#ifdef SWIG
+
+  %extend
+  {
+    std::vector<tesseract_planning::Instruction> getInstructions()
+    {
+      std::vector<tesseract_planning::Instruction> o;
+      std::copy($self->begin(), $self->end(), std::back_inserter(o));
+      return o;
```

Comment:
Could this return container_, it should copy it? Also this be exposed in the c++ interface?

Reply:
The swig wrappers can't handle std::reference_wrapper properly. This change copies the data into a normal std::vector<std::instruction> type to return to Python.

Reply:
There is not any benefit to having this return type in C++

Reply:
Ah, ok.

Reply:
Oh, sorry, this is a different file than I realized. Yeah, having these in C++ might be useful. It could be a reference.

Reply:
OK

Reply:
Updated both to be C++ functions